src/PowerShellEditorServices/Workspace/Workspace.cs

```diff
@@ -650,50 +650,77 @@ public static string ConvertPathToDocumentUri(string path)
             }
 
             string escapedPath = Uri.EscapeDataString(path);
-            var docUriStrBld = new StringBuilder(escapedPath);
+
+            // Max capacity of the StringBuilder will be the current escapedPath length
+            // plus extra chars for file:///.
+            var docUriStrBld = new StringBuilder(escapedPath.Length + fileUriPrefix.Length + 3);
+            docUriStrBld.Append(fileUriPrefix).Append("//");
 
             if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
             {
-                // VSCode file URIs on Windows need the drive letter lowercase.
-                // Search original path for colon since a char search (no string culture involved)
-                // is faster than a string search.
+                // VSCode file URIs on Windows need the drive letter to be lowercase. Search the
+                // original path for colon since a char search (no string culture involved) is
+                // faster than a string search.  If found, then lowercase the associated drive letter.
                 if (path.Contains(':'))
```

Comment:
Could a UNC path have a colon in it? I'm pretty sure Windows paths can't have colons in them (except for `C:` etc)?

Reply:
I don't think so but I'm not 100% sure.  But if a path does **not** contain a colon it doesn't necessarily mean it is a UNC path.  It could be relative path which would not have that third `/` that indicates an absolute path.

Reply:
Ah... I understand. Thanks!